Reporting/M5 Document/Requirement Update.md

```diff
@@ -0,0 +1,79 @@
+        > seamless communication and interaction among users. The core
+        > functionality revolves around instant messaging, enabling
+        > users to connect with friends and communities effortlessly.
+        > Our endeavor is to construct a robust full-stack web
+        > application utilizing cutting-edge technologies to ensure a
+        > superior user experience.
+
+-   [A list of user groups for your software, along with an example
+    > scenario for each user group of how they will use the
+    > software]{.underline}
+
+    -   **UBCO Students**
+
+        -   **Scenario:** Colin, a UBCO student, utilizes Accord to
+            > connect with classmates, collaborate on group projects,
+            > and engage with campus communities. He participates in
+            > discussion servers dedicated to his courses, where he
+            > seeks clarification on assignments, shares study
+            > materials, and discusses course concepts with peers.
+
+    -   **Community Owners/Admin:**
+
+        -   **Scenario:** Immanuel, a community admin for his Helldivers
+            > game group, utilizes Accord to create a vibrant online
+            > space for gamers. He establishes discussion
+            > servers/channels dedicated to specific activities and
+            > topics in games. Immanuel moderates discussions, organizes
+            > events, and ensures a welcoming environment for all
+            > community members.
+
+    -   **Casual users**
+
+        -   **Scenario:** Toby, a casual user of Accord, utilizes Accord
+            > to connect with distant relatives and old friends. He
+            > occasionally joins group chats to catch up with
+            > acquaintances and share updates on his life. While Toby
+            > may not engage with the platform as frequently as other
+            > users, Accord provides him with a convenient and
+            > accessible means of communication.
+
+-   [The final list of requirements of the software that was built.
+    > Incorporate feedback from M2.]{.underline}
+
+    -   Allow users to create an account on the platform.
+
+    -   Allow the user to log in to the platform with an existing
+        > account.
+
+    -   Allow the user to log out of their account.
+
+    -   Allow the user to edit their account (username, password, email,
+        > etc.).
+
+    -   Allow the user to send text messages.
+
+    -   Allow the user to receive text messages.
+
+    -   Allow the user to add another user as a friend.
+
+    -   Allow users to remove another user from their friend list.
+
+    -   Allow the user to create a group chat.
+
+    -   Allow group chat owners to manage a group chat.
+
+    -   Allow server owners to create a text channel.
+
+    -   Allow server owners to edit text channels.
+
+    -   Allow users to add other users to the server.
+
+    -   Allow server owners to delete a message from a text channel.
```

Comment:
Change to "Allow text channel owners to have the ability to delete all messages from a text channel" (this is currently not implemented yet but should be fast)